taskfile.yaml

```diff
@@ -144,17 +144,21 @@ tasks:
       - task: deploy
       - sh hack/setup.sh
 
-  fast-e2e-test:
+  run-e2e-test-suite:
```

Comment:
when would you run the test without deploying the operator? You will always need to use the `fast-e2e-test` target. It doesn't make sense to have a separate target for just running the test that is doesn't have `internal: true`

Reply:
- to deploy the operator once, and run the test multiple times
- to run the test after the operator was already deployed earlier

Reply:
why would you want to run the same tests multiple times? I understand that we need to run it, but exposing this complicates things.

Reply:
For testing and debugging the test.

Reply:
iiuc, you're saying you would make a change to the test while keeping the code the same and want to build and test the test only?